bc_obps/registration/models/bcghg_operation_or_facility.py

```diff
@@ -0,0 +1,53 @@
+from registration.models.user import User
+import re
+import typing
+from common.models import BaseModel
+from django.db import models
+from registration.constants import BCGHG_ID_REGEX
+from simple_history.models import HistoricalRecords
+from django.utils import timezone
+from django.core.exceptions import ValidationError
+
+
+class BcghgOperationOrFacility(BaseModel):
```

Comment:
Didn't know what to call this, the boro id one is called bc_obps_regulated_operation